Generation fails without error message for invalid column type

**Steps to reproduce:**

1. Create a `.sq` spec that includes an invalid column type:

```sql
CREATE TABLE categories (
  id STRING NOT NULL PRIMARY KEY,
  title String NOT NULL
);
```

1. Build.

**Expected Result:**

Build fails with error message indicating that the column type is invalid.

**Actual Result:**

Build fails with no error message. (<- literally 'no error message')

**Output**

```
→ ./gradlew :app:generateDebugSqlDelightInterface
Incremental java compilation is an incubating feature.
:app:generateDebugSqlDelightInterface FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:generateDebugSqlDelightInterface'.
> java.lang.NullPointerException (no error message)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 2.259 secs
```

**Stacktrace:**

```
Caused by: java.lang.NullPointerException
        at com.squareup.sqldelight.model.ColumnKt.getSqliteClassName(Column.kt:64)
        at com.squareup.sqldelight.model.ColumnKt.getJavaType(Column.kt:74)
        at com.squareup.sqldelight.types.Resolver.resolve(Resolver.kt:257)
```

I believe this is caused by `Column.kt:59`, which tries to treat the unknown column type as a string literal and is unprotected.

```kotlin
val className = javaTypeName.STRING_LITERAL().text.trim('\'')
```

--- a/sqldelight-compiler/src/main/antlr/com/squareup/sqldelight/Sqlite.g4
+++ b/sqldelight-compiler/src/main/antlr/com/squareup/sqldelight/Sqlite.g4
@@ -660,10 +660,10 @@ K_JAVA_DOUBLE: D 'o' 'u' 'b' 'l' 'e';
 K_JAVA_STRING: S 't' 'r' 'i' 'n' 'g';
 K_JAVA_BYTE_ARRAY: 'b' 'y' 't' 'e' '[' ']';
 
-K_INTEGER: I N T E G E R;
-K_REAL: R E A L;
-K_TEXT: T E X T;
-K_BLOB: B L O B;
+K_INTEGER: 'I' 'N' 'T' 'E' 'G' 'E' 'R';
+K_REAL: 'R' 'E' 'A' 'L';
+K_TEXT: 'T' 'E' 'X' 'T';
+K_BLOB: 'B' 'L' 'O' 'B';
 
 SCOL : ';';
 DOT : '.';

--- a/sqldelight-gradle-plugin/src/test/fixtures/invalid-column-type/failure.txt
+++ b/sqldelight-gradle-plugin/src/test/fixtures/invalid-column-type/failure.txt
@@ -1,0 +1,1 @@
+Test.sq line 3:8 - mismatched input 'Text' expecting {K_INTEGER, K_REAL, K_TEXT, K_BLOB}

--- a/sqldelight-gradle-plugin/src/test/fixtures/invalid-column-type/src/main/sqldelight/com/sample/Test.sq
+++ b/sqldelight-gradle-plugin/src/test/fixtures/invalid-column-type/src/main/sqldelight/com/sample/Test.sq
@@ -1,0 +1,4 @@
+CREATE TABLE categories (
+  id TEXT NOT NULL PRIMARY KEY,
+  title Text NOT NULL
+);

